docs/source/quickstart.rst

```diff
@@ -28,21 +28,19 @@ Code
 
     @trainer.on(Events.ITERATION_COMPLETED(every=log_interval))
     def log_training_loss(trainer):
-        print("Epoch[{}] Loss: {:.2f}".format(trainer.state.epoch, trainer.state.output))
+        print(f"Epoch[{trainer.state.epoch}] Loss: {trainer.state.output:.2f}")
 
     @trainer.on(Events.EPOCH_COMPLETED)
     def log_training_results(trainer):
         evaluator.run(train_loader)
         metrics = evaluator.state.metrics
-        print("Training Results - Epoch: {}  Avg accuracy: {:.2f} Avg loss: {:.2f}"
-              .format(trainer.state.epoch, metrics["accuracy"], metrics["nll"]))
+        print(f"Training Results - Epoch: {trainer.state.epoch}  Avg accuracy: {metrics["accuracy"]:.2f} Avg loss: {metrics["nll"]:.2f}")
 
     @trainer.on(Events.EPOCH_COMPLETED)
     def log_validation_results(trainer):
         evaluator.run(val_loader)
         metrics = evaluator.state.metrics
-        print("Validation Results - Epoch: {}  Avg accuracy: {:.2f} Avg loss: {:.2f}"
-              .format(trainer.state.epoch, metrics["accuracy"], metrics["nll"]))
+        print("Validation Results - Epoch: {trainer.state.epoch}  Avg accuracy: {metrics["accuracy"]:.2f} Avg loss: {metrics["nll"]:.2f}")
```

Comment:
```suggestion
        print("Validation Results - Epoch: {trainer.state.epoch}  Avg accuracy: {metrics['accuracy']:.2f} Avg loss: {metrics['nll']:.2f}")
```

Reply:
It looks like line 43 is missing the 'f' after the print and before the "Validation